robot/rospackages/src/image_scp/config.py

```diff
@@ -0,0 +1,6 @@
+#enter host computer address
+host = "HOST_URL"
+#enter user name for host computer
+user = "USER NAME FOR HOST"
+#enter desired image path on host computer
+remote_path_1 = "PATH FOR IMAGE ON HOST COMPUTER"
```

Comment:
So what happens if we want to dynamically send an image. I want to call this script lets say, everytime I take a new picture. This picture has a new name. Perhaps you want to just put a folder that needs to be synced. But then we will have issues if there's leftover files.